cardano-api/src/Cardano/Api/Script.hs

```diff
@@ -835,6 +871,7 @@ fromAllegraTimelock timelocks = go
 
 instance ToJSON (Script lang) where
   toJSON (SimpleScript _ script) = toJSON script
+  toJSON (PlutusScript _ _) = Aeson.Null
```

Comment:
Perhaps better to leave this as an `error "TODO ..."` rather than `Null`. The latter is too easy to forget or miss in a `grep TODO`.